recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "jupyter_core" %}
-{% set version = "5.3.0" %}
+{% set version = "5.5.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 6db75be0c83edbf1b7c9f91ec266a9a24ef945da630f3120e1a0046dc13713fc
+  url: https://github.com/jupyter/jupyter_core/releases/download/v{{ version }}/jupyter_core-{{ version }}.tar.gz
```

Comment:
why not taking from pypi ? looks 5.5.0 is there https://pypi.org/project/jupyter-core/

Reply:
I'm guessing because the tests moved. They used to be available as `jupyter_core.tests` but are now at at the root. @cbouss we can still use the sdist since it contains the tests, see https://inspector.pypi.io/project/jupyter-core/5.5.0/packages/5c/3d/c75bda485eaf15cd430383deb0c441aa822679ea88c5b32cfc2013f678e1/jupyter_core-5.5.0.tar.gz/.